### PR TITLE
Add sabnzbd exporter to docs

### PR DESCRIPTION
**Pull request by msroest**
Adding new SABnzbd (https://sabnzbd.org) exporter to documentation

### CODE REVIEW

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -149,6 +149,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [Transmission exporter](https://github.com/metalmatze/transmission-exporter)
    * [Unbound exporter](https://github.com/kumina/unbound_exporter)
    * [Xen exporter](https://github.com/lovoo/xenstats_exporter)
+   * [SABnzbd exporter](https://github.com/msroest/sabnzbd_exporter)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: Keep it sorted.

**Comment by msroest** [comment]
Cool I'll make the changes tonight.  Thanks

**Comment by brian-brazil** [comment]
Thanks!

**Comment by msroest** [comment]
Or I'll do it right now :).  Default port added to the python (it was set in the dockerfile as a default but not in the code) and metric renamed

**Pull request merged**